extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -73,7 +68,52 @@ const LightboxControlsModes = {
   CONTROLS_HIDDEN: 0,
 };
 
-const SWIPE_TO_CLOSE_THRESHOLD = 10;
+/**
+ * The number of pixels of movement to go from the darkest to lightest overlay
+ * while doing a swipe to close gesture.
+ */
+const SWIPE_TO_CLOSE_DISTANCE = 200;
+/**
+ * The number of pixels needed to close when doing a swipe to close gesture.
+ */
+const SWIPE_TO_CLOSE_DISTANCE_THRESHOLD = SWIPE_TO_CLOSE_DISTANCE / 4;
+/**
+ * The number of pixels needed to completely fade out the controls when doing a
+ * swipe to close gesture.
+ */
+const SWIPE_TO_HIDE_CONTROLS_DISTANCE = SWIPE_TO_CLOSE_DISTANCE / 4;
+/**
+ * The velocity at which to close light box from a swipe, regardless of distance
+ * travelled.
+ */
+const SWIPE_TO_CLOSE_VELOCITY_THRESHOLD = 0.65;
+/**
+ * The lowest opacity for the background and controls when doing swipe to close
+ * gesture.
+ */
+const SWIPE_TO_CLOSE_MIN_OPACITY = 0.2;
+/** The smallest scale possible when doing swipe to close gesture. */
+const SWIPE_TO_CLOSE_MIN_SCALE = 0.85;
+/**
+ * How much distance to cover, based on the velocity, when a user releases a
+ * swipe to close gesture.
+ */
+const SWIPE_TO_CLOSE_VELOCITY_TO_DISTANCE_FACTOR = 22.5;
+/**
+ * How much time to spend, based on the distance to travel, when moving to the
+ * final location of a swipe (after the user has released).
+ */
+const SWIPE_TO_CLOSE_DISTANCE_TO_TIME_FACTOR = 1;
+/**
+ * How much time to spend, based on the distance to travel, when snapping back
+ * after an cancelled swipe to close gesture.
+ */
+const SWIPE_TO_CLOSE_SNAP_BACK_TIME_FACTOR = 5;
+/**
+ * The timing function to use when carrying momentum after releasing a swipe to
+ * close gesture. This closely approximates an expontential decay of velocity.
+ */
+const SWIPE_TO_CLOSE_MOMENTUM_TIMING = 'cubic-bezier(0.15, .55, .3, 0.95)';
 
 // Use S Curves for entry and exit animations
 const TRANSITION_CURVE = {x1: 0.8, y1: 0, x2: 0.2, y2: 1};
```

Comment:
Also since we tweaked numbers, the following numbers are fair game too. In fact `MAX_DISTANCE_APPROXIMATION` of 250px is probably a really bad approximation for certain devices, since the max distance an image can travel varies based on the screen size.

Reply:
Perhaps it should be based on viewport width/height (maybe the smaller of the two?).

Reply:
Okay, I think I'm good here.

The one change I feel strongly about is changing `MAX_TRANSITION_DURATION` to 700ms, as we discussed.

Otherwise, ship it. :)